sanitize_html/lib/sanitize_html.dart

```diff
@@ -61,14 +60,10 @@ String sanitizeHtml(
   bool Function(String) allowElementId,
   bool Function(String) allowClassName,
 }) {
-  final doc = runZoned(() {
-    return html.DocumentFragment.html(
-      htmlString,
-      validator: SaneHtmlValidator(
-        allowElementId: allowElementId,
-        allowClassName: allowClassName,
-      ),
-    );
+  return runZoned(() {
```

Comment:
Yeah, we can remove `runZoned`, it was only used to deal with some print statements in `package:universial_html`